src/components/home/components/Carousel.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { Carousel } from 'antd';
+import { CAROUSEL_DATA } from '../constants';
+
+//this is the carousel component
+class CarouselImages extends React.Component {
+    render() {
+        return (
+            <Carousel effect="fade" autoplay>
+                {CAROUSEL_DATA.map((data, index) => (
+                    <div key={index}>
+                        <img src={data.src} />
+                        <div className="carousel-text">
+                            <h3>{data.text}</h3>
+                        </div>
+                    </div>
+                ))}
+            </Carousel>
+        );
+    }
+}
+
+export default CarouselImages;
```

Comment:
remove this code from this PR. This will cause conflicts.
remove all that concerns Carousel